File: aula03-managed-point-java/Ponto.java
import java.lang.Math;

class A{
    static class X{}
}

class B{}


public class Ponto {
    public int w, x, y, z;

    public Ponto(int x, int y)
    {
        this.x = x;
        this.y = y;
    }

    public double getModule() {
        return Math.sqrt((double)x*x + y*y);
    }

    public void print(){
        System.out.println(
            String.format("Print 3 super (x = %d, y = %d)", x, y));
    }
}

File: aula03-managed-point-cs/Ponto.cs
using System;

class A{
    public class X{}
}

class B{
    public B(){ new A.X();}
}


public class Ponto {
    public int x, y, z;

    public Ponto(int x, int y)
    {
        this.x = x;
        this.y = y;
    }

    public double getModule() {
        return Math.Sqrt(x*x + y*y);
    }

    public void print(){
        Console.WriteLine(
            String.Format("Print 3 super (x = {0}, y = {1})", x, y));
    }
}